### ISSUE TITLE
Bazel apt repo is not installing 1.1.0

### ISSUE THREAD

**Issue by aaliddell** [bazelbuild/bazel#10199]
### Description of the problem / feature request:

When installing Bazel via apt, the latest version available appears to be 1.0.1 (after an update and upgrade):

```
apt-cache show bazel    
---                                                                                                                                                                                                                                                   
Package: bazel
Version: 1.0.1
Architecture: amd64
Depends: g++, zlib1g-dev, bash-completion, unzip, python
Suggests: google-jdk | java8-sdk-headless | java8-jdk | java8-sdk |
 oracle-java8-installer
Maintainer: The Bazel Authors <bazel-dev@googlegroups.com>
Priority: optional
Section: contrib/devel
Filename: pool/jdk1.8/b/bazel/bazel_1.0.1_amd64.deb
Size: 43801306
SHA256: 6b9cf2039f57dcf7fe8bc91e58c10f8bd5be49ae628341be08609d1c99b46148
SHA1: 04a2bbde9b679171c43ad84838a1bc33c41ee728
MD5sum: dad0fcf4e2de8d9e96eadd5f655e8757
Description: Bazel is a tool that automates software builds and tests.
 Supported build tasks include running compilers and linkers to produce
 executable programs and libraries, and assembling deployable packages
 for Android, iOS and other target environments. Bazel is similar to
 other tools like Make, Ant, Gradle, Buck, Pants and Maven.
Description-md5: 72cec041981bfe49fb7d7b6d540d9076
Homepage: http://bazel.build
Built-Using: bazel (HEAD)
```

The 1.1.0 .deb is available in the repo, but doesn't seem to be used: `pool/jdk1.8/b/bazel/bazel_1.1.0_amd64.deb`

### What operating system are you running Bazel on?

Ubuntu 18.04
Line in `/etc/apt/sources.list.d/bazel.list` is `deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8`

**Comment by meisterT** [comment]
cc @meteorcloudy

**Comment by meteorcloudy** [comment]
@philwo is working on our apt repo now.

**Comment by philwo** [comment]
That's because 1.0.1 was released after 1.1.0, so the apt repository generation script of that release overwrote the metadata to say that 1.0.1 is the latest version 😬

This will be fixed as part of the apt repository generation rework.

**Comment by philwo** [comment]
@meteorcloudy fixed this in https://github.com/bazelbuild/bazel/commit/31627057454717d2216da0791969207569a1f423. :) Our apt repository now contains all stable version released since 1.0.0 and will continue to do so in the future, so `apt-get install` should correctly install the highest available version (and not the one that happened to be released last).
